flink/src/test/java/org/apache/iceberg/flink/FlinkCatalogTestBase.java

```diff
@@ -119,6 +123,12 @@ protected String warehouseRoot() {
     }
   }
 
+  protected String getFullQualifiedTableName(String tableName) {
+    final List<String> levels = new ArrayList<>(Arrays.asList(icebergNamespace.levels()));
```

Comment:
Nit: in apache iceberg, we usually use the unified `Lists.newArrayLists` to create a new ArrayList.

Reply:
will fix